pallets/foreign-investments/src/tests.rs

```diff
@@ -1458,12 +1625,13 @@ mod redemption {
 				Some(RedemptionInfo {
 					foreign_currency: FOREIGN_CURR,
 					swapped_amount: 0,
-					collected: CollectedAmount::default(),
+					collected_tranche_tokens: 0,
```

Comment:
I don't understand why this should be 0. First, we process 3/4 of the redemption and then collect it. So why would it be 0?

Reply:
We process 3/4 of the total amount. So, the total possible collectable amount is 3/4.
We later make two swaps 1/2 and 1/4, so after both, we no longer have any pending tranche tokens to notify to the user.

Note that the pending 1/4 amount is not collected. If the user wants those funds, they need to cancel the redemption or wait until they are collected.